ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/search_tree/algorithms/Leaf.java

```diff
@@ -92,11 +92,12 @@ public enum Status {
          Network network,
          Set<NetworkAction> alreadyAppliedNetworkActionsInPrimaryState,
          NetworkActionCombination newCombinationToApply,
+         RangeActionActivationResult prePerimeterActivationResult,
```

Comment:
add a comment explaining why sometimes prePerimeterActivationResult will be different than new RangeActionActivationResultimpl(preperimeterSetpoints)

Reply:
I renamed it with its old name because it was more precise so i consider this conversation resolved